### PR TITLE
debatcher updates for cosmos db #HL7-2456

### PR DESCRIPTION
**Pull request by emilio32-lang**
cosmos db updates
updates for event hubs/output bindings
removing eventHunSender

### CODE REVIEW

File: fns-hl7-pipeline/fn-receiver-debatcher/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>gov.cdc.dataexchange</groupId>
     <artifactId>receiver-debatcher</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>${fn.version}</version>
     <packaging>jar</packaging>
```
Comment (marcia-schulman) [inline fns-hl7-pipeline/fn-receiver-debatcher/pom.xml:10]: We are doing the version differently now -- please see the Comments section of the Jira ticket HL7-2454

Comment (emilio32-lang) [reply fns-hl7-pipeline/fn-receiver-debatcher/pom.xml:10]: updated, please review

Comment (emilio32-lang) [reply fns-hl7-pipeline/fn-receiver-debatcher/pom.xml:10]: updated, please review

File: fns-hl7-pipeline/fn-receiver-debatcher/pom.xml
[the remark this answers is not in the document]
Comment (emilio32-lang) [reply fns-hl7-pipeline/fn-receiver-debatcher/pom.xml:1]: updated, please review

**Pull request merged**